server/driver/wivrn_session.cpp

```diff
@@ -162,13 +167,44 @@ xrt_result_t xrt::drivers::wivrn::wivrn_session::create_session(xrt::drivers::wi
 	if (self->hmd->face_tracking_supported)
 		usysds->base.base.static_roles.face = self->hmd.get();
 
-	if (self->left_hand)
+	xrt_device * active_left_hand = nullptr;
+	xrt_device * active_right_hand = nullptr;
+
+#ifdef WIVRN_FEATURE_STEAMVR_LIGHTHOUSE
+	auto use_steamvr_lh = std::getenv("WIVRN_USE_STEAMVR_LH");
+	xrt_system_devices * lhdevs = NULL;
+
+	if (use_steamvr_lh && steamvr_lh_create_devices(&lhdevs) == XRT_SUCCESS)
+	{
+		U_LOG_W("Found %lu lighthouse devices:", lhdevs->xdev_count);
```

Comment:
This should be info level

Reply:
just removed it, Monado logs them as well